addons/launcher-template/launcher-template.addon

```diff
@@ -0,0 +1,4 @@
+# Name: launcher-template
```

Comment:
Probably the template needs to be updated to remove any reference to `launchpad`. That's why I was pointing to the URL before